<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuildingsForm_template</class>
 <widget class="QDialog" name="BuildingsForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="The dialog to list all buildings">Buildings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QListWidget" name="buildingsListWidget"/>
     <widget class="QPlainTextEdit" name="currentBuildingTextEdit"/>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addBuildingPushButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="modifyBuildingPushButton">
       <property name="text">
        <string>Modify</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeBuildingPushButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="moveBuildingUpPushButton">
       <property name="text">
        <string comment="It means move the item up in the list">Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveBuildingDownPushButton">
       <property name="text">
        <string comment="It means move the item down in the list">Down</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="commentsPushButton">
       <property name="text">
        <string>Comments</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="sortBuildingsPushButton">
       <property name="text">
        <string>Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
